extractors/bookseries/karelians/extractors/tests/migrations/test_location_extraction.py

```diff
@@ -184,7 +184,7 @@ def should_extract_short_place_names_if_they_are_in_white_list(
         results, metadata = finnish_extractor.extract(
             {'text': LOCATION_HEURISTICS['short_white_listed_name']['text']}, {}, {}
         )
-        locations = results['finnishLocations'][KEYS["otherlocations"]]
+        locations = results['finnishLocations'][KEYS['otherlocations']]
```

Comment:
Hmm, mites nää muuttuu tässä flaken myötä? Eikös sen blackin pitäisi nämä muuttaa? Vai onko tässä vaan mennyt blackin tekemät formatoinnit konffauksen jälkeen vahingossa tähän committiin? No ehkä se ja sama. Tai noh, jos haluaa hifistellä niin varmaan blackin voi ajaa siinä aiemmassa commitissa rebasen aikana niin ehkä nää häviää automaagisesti tästä commitista?

Reply:
Niin siis mä tein kokonaan uusiksi sen `black`-formatointicommitin. Annoin sille sen tarvitseman option, että se skippaa string normalisaation, ja sen option kanssa uusi `black`-formatointicommitti on tehty. `black`ia ei voi konfiguroida toimimaan niin, että se pakottaa hipsut yksöishipsuiksi. `black` voi joko muuttaa kaikki hipsut tuplahipsuiksi, tai `black`in voi conffata skippaamaan string normalisaation kokonaan. Nämä ovat vaihtoehdot :D Tämän takia se on nyt `flake8`:n vastuulla kytätä hipsuja.